### ISSUE TITLE
A tool for exploring pointers is missing

### ISSUE THREAD

**Issue by tesonep** [guillep/pharo-core#10681]
In Pharo10 there is not replacement for PointerExplorer.
We can integrate RefHunter or anything else, but it is a needed tool.
https://github.com/pavel-krivanek/RefsHunter

**Comment by astares** [comment]
From the UI side it would be nice if the tool would not be a separate one but (lazily) available from inspector.

**Comment by kasperosterbye** [comment]
I have been able to use `ReferenceFinder` with great success, but I assume the tools you refer to are better?

**Comment by Ducasse** [comment]
What torsten is saying is what pablo wants :)

**Comment by tesonep** [comment]
Yes, the ReferenceFinder works great, I think we can have a nice UI for it.

**Comment by pavel-krivanek** [comment]
ReferenceFinder is much better than RefsHunter (it is much slower and consumes huge amount of memory. It only handles references from contexts better).
